architecture-building-systems/CityEnergyAnalyst
# 2769 document release

This PR improves the documentation page **How to create a new release** 
To test:
1. switch to the branch, and open `\CityEnergyAnalyst\docs\_build\html\index.rst`
2. find the link to **How to create a new release** 
3. go through the steps and check if those are accurate
